app/services/events/stores/postgres/unique_count_query.rb

```diff
@@ -96,16 +126,32 @@ def breakdown_query
         def events_cte_sql
           # NOTE: Common table expression returning event's timestamp, property name and operation type.
           <<-SQL
-            WITH events_data AS (
-              (#{
-                events
-                  .select(
-                    "timestamp, \
-                    #{sanitized_property_name} AS property, \
-                    COALESCE(events.properties->>'operation_type', 'add') AS operation_type",
-                  ).to_sql
-              })
-            )
+            WITH events_data AS (#{
+              events
+                .select(
+                  "timestamp, \
+                  #{sanitized_property_name} AS property, \
+                  COALESCE(events.properties->>'operation_type', 'add') AS operation_type",
+                ).to_sql
+            })
+          SQL
+        end
+
+        def grouped_events_cte_sql
+          groups = store.grouped_by.map.with_index do |group, index|
+            "#{sanitized_property_name(group)} AS g_#{index}"
+          end
+
+          <<-SQL
+            WITH events_data AS (#{
+              events
+                .select(
+                  "#{groups.join(', ')}, \
+                  timestamp, \
+                  #{sanitized_property_name} AS property, \
+                  COALESCE(events.properties->>'operation_type', 'add') AS operation_type"
```

Comment:
Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call. (https://rubystyle.guide#no-trailing-params-comma)